framework/src/org/checkerframework/framework/flow/CFAbstractTransfer.java

```diff
@@ -115,6 +115,16 @@ public CFAbstractTransfer(CFAbstractAnalysis<V, S, T> analysis) {
         this.sequentialSemantics = !analysis.checker.hasOption("concurrentSemantics");
     }
 
+    // Note that the user can always forcefully turn on concurrent semantics via -AconcurrentSemantics.
```

Comment:
I don't get the point of this `note`. That comment equally applies to the existing constructor. It might be best to provide javadoc.